src/SmartComponents/Advisor/StackChart.js

```diff
@@ -34,44 +34,45 @@ const StackChart = ({ data, intl }) => {
     const labelComponent = () => <ChartTooltip text={ ({ datum }) => `${datum.name}: ${datum.y}` } constrainToVisibleArea />;
 
     return <React.Fragment>
-        <Chart
-            ariaDesc='Advisor recommendations by severity'
-            ariaTitle='Advisor recommendations by severity'
-            padding={ { left: 0, right: 0, bottom: 56, top: 20 } }
-            width={ 500 }
-            legendPosition='bottom-left'
-            height={ 110 }
-            maxWidth={ 600 }
-            legendComponent={ <ChartLegend
-                data={ legendData }
-                width={ 400 }
-                events={ [{
-                    target: 'labels', eventHandlers: {
-                        onClick: legendClick,
-                        onMouseOver: () => {
-                            return [{
-                                mutation: (data) => {
-                                    return {
-                                        style: Object.assign({}, data.style, { cursor: 'pointer' })
-                                    };
-                                }
-                            }];
+        <div style={ { maxWidth: 600 } }>
```

Comment:
Not sure what this change is seeking to accomplish... without screen shots or text explaining the what and the why, gotta big time 🙅‍♂ this

Reply:
@AllenBW this makes it look like I replaced the whole component, but I only modified: height, width and padding bottom, so that the chart would closely resemble the ratio on Kyle's mocks

Reply:
so true! it does, n i know, the diff isn't that big, but all the same, as a convenience for reviewers, showing visual changes (screenshot) would be most appreciated

Reply:
i agree! @AllenBW  I've been having problems with the advisor card loading, so i wasn't able to take a screenshot, so just removed the changes for now